packages/wesley-cli/src/commands/models.mjs

```
@@ -28,16 +24,13 @@ export class ModelsCommand extends GeneratorCommand {
 
     const result = await generator.generate(ir, { outDir: options.outDir });
 
-    if (!options.quiet) {
-      console.log('✨ Generated model classes:');
-      result.files.forEach((file) => console.log(`  ✓ ${file}`));
-      console.log(`\n📁 Target: ${result.target} (${result.outputDir})`);
+    if (!options.quiet && !options.json) {
+      logger.info('Generated model classes:');
+      result.files.forEach((file) => logger.info(`  ${file}`));
```

Biome (2.4.4)

[error] This callback passed to forEach() iterable method should not return a value.

(lint/suspicious/useIterableCallbackReturn)